recipes/astral/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "astral" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ab0c08f2467d35fcaeb7bad15274743d3ac1ad18b5391f64a0058a9cd192d37d
+
+build:
+  #noarch: python
```

Comment:
I think this should probably be a `noarch` package.

Reply:
Indeed! I commented it out to test if it was responsible for the failure, which we now know is due an upstream conda-build issue. I'll uncomment this and rebuild after that is fixed (and address any real build issues that may come up!)